packages/0x.js/scripts/postpublish.js

```diff
@@ -0,0 +1,43 @@
+const execAsync = require('async-child-process').execAsync;
+const postpublish_utils = require('../../../scripts/postpublish_utils');
+const packageJSON = require('../package.json');
+
+const cwd = __dirname + '/..';
+const subPackageName = packageJSON.name;
+
+let tag;
+let version;
+postpublish_utils.getLatestTagAndVersionAsync(subPackageName)
+    .then(function(result) {
+        console.log('POSTPUBLISH: Releasing...');
```

Comment:
do you still want to keep the console.log statements in here?

Reply:
Not this one, but the others yes. I think it's helpful to know where something went wrong.